.github/workflows/ci.yml

```diff
@@ -6,6 +6,12 @@ jobs:
   test:
     name: Test
     runs-on: ubuntu-latest
+    container:
+      image: node:18
```

Comment:
this makes the test run in a container.. which then has access to the services using their name as the host, e.g. `http://elastic7:9200`